### PR TITLE
Handling package dependencies in infiniband tests

### PR DESCRIPTION
**Pull request by narasimhan-v**
Handled dependencies for ssh and iputils packages for different
distros for all infiniband tests.

Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

### CODE REVIEW

File: io/net/infiniband/ib_bw_perf.py [hunk 1]
```diff
@@ -40,10 +40,17 @@ def setUp(self):
...
-            if not sm.check_installed(pkg) and not sm.install(pkg):
+        smm = SoftwareManager()
+        detected_distro = distro.detect()
+        pkgs = ["perftest"]
+        if detected_distro.name == "Ubuntu":
+            pkgs.append('openssh-client')
+        elif detected_distro.name == "SuSE":
+            pkgs.append('openssh')
+        else:
+            pkgs.append('openssh-clients')
```
Comment (vrbagalkote) [inline io/net/infiniband/ib_bw_perf.py]: is it not better to check specifically with Red-Hat/Rhel, rather than executing in else.

Comment (narasimhan-v) [inline io/net/infiniband/ib_bw_perf.py]: Yes, but the package name in `rhel` is also `openssh-clients`.

**Comment by PraveenPenguin** [comment]
@vrbagalkote thanks for review

**Comment by narasimhan-v** [comment]
@PraveenPenguin Thanks

**Pull request merged**